packages/wrangler/src/config/validation-helpers.ts

```diff
@@ -231,6 +231,27 @@ export const isString: ValidatorFn = (diagnostics, field, value) => {
 	return true;
 };
 
+/**
+ * Validate that the field is a non-empty string.
+ */
+export const isNonEmptyString: ValidatorFn = (diagnostics, field, value) => {
+	if (value !== undefined && typeof value !== "string") {
+		diagnostics.errors.push(
+			`Expected "${field}" to be of type string but got ${JSON.stringify(
+				value
+			)}.`
+		);
+		return false;
+	}
```

Comment:
Could this call the existing `isString()` validation function? It seems like the same logic

Reply:
✅